### PR TITLE
Completed Slicing Exercise in Lesson 3

### PR DESCRIPTION
**Pull request by josh1209**
Finished assert statements and used simplified code in all functions.

### CODE REVIEW

File: students/Josh_HOff/lesson03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+#    print(fruits)
+#bonus for series 2
+fruits = (fruits * 2)
+x = 1
+while x == 1:
+    response = input('Please delete a fruit: ')
+    for i in fruits:
+        if i == response:
+            fruits.remove(response)
+            x = 2
```
Comment (natasha-aleksandrova) [inline students/Josh_HOff/lesson03/list_lab.py]: the `x` seems mysterious 🙂 

if you would like to exit on the remove condition then this is a better pattern:
```
while True:
    ....
    if fruit == response:
        ...
        break
```
use `break`

File: students/Josh_HOff/lesson03/list_lab.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+#series 3
+for i in fruits3:
+    x = 1
+    response = input('Do you like {}? '.format(i))
+    if response != 'yes' and response != 'no':
+        while x == 1:
+            response = input('Please choose "yes" or "no": ')
+            if response == 'yes' or response == 'no':
+                x = 2
```
Comment (natasha-aleksandrova) [inline students/Josh_HOff/lesson03/list_lab.py]: same here about the while loop pattern.

**Comment by josh1209** [comment]
Cool, cool. I'll use that style for while loops in the future.

**Comment by natasha-aleksandrova** [comment]
sounds good!

**Pull request merged**